### DISCUSSION TITLE
How can I use my own genome (non-model organisms)?

### DISCUSSION THREAD

**Discussion by kuangzhuoran** [kcleal/gw#57] [Q&A]
The manual seems to default to selecting genome only for model species.

**Comment by kuangzhuoran** [comment]
Well, I edit the .gw.ini and add my genome (named saker) into this file

Then I run gw saker, and I get:
Fontconfig error: Cannot load default config file: No such file: (null)

█▀▀ █ █ █
█▄█ ▀▄▀▄▀
GLFW Error: GLX: No GLXFBConfigs returned
GLFW Error: GLX: Failed to find a suitable GLXFBConfig
ERROR: glfwCreateWindow failed



Next I run GW_DEBUG=1 gw saker, and I get:
Fontconfig error: Cannot load default config file: No such file: (null)

█▀▀ █ █ █
█▄█ ▀▄▀▄▀
DISPLAY=localhost:15.0
LIBGL_DEBUG=
LIBGL_ALWAYS_INDIRECT=
LIBGL_ALWAYS_SOFTWARE=
MESA_DEBUG=
MESA_GL_VERSION_OVERRIDE=
GLX_DEBUG=
GALLIUM_DRIVER=
__GL_LOG_LEVEL=
LD_DEBUG=
XDG_SESSION_TYPE=
WAYLAND_DEBUG=
GLFW version: 3.4.0 Wayland X11 GLX Null EGL OSMesa monotonic shared
GLFW_CONTEXT_VERSION_MAJOR=
GLFW_CONTEXT_VERSION_MINOR=
GLFW Error: GLX: No GLXFBConfigs returned
GLFW Error: GLX: Failed to find a suitable GLXFBConfig
ERROR: glfwCreateWindow failed

**Reply by kcleal** [reply]
This error suggests a problem with graphics drivers/libs - gw is unable to correctly load OpenGL. You can try setting `LIBGL_DEBUG=1 GW_DEBUG=1 gw hg38`, or run `glxinfo | grep "OpenGL version"`. If this is a conda instal, it might be resolved by installing gw from source.

Alternatively, if you have already built gw from soure then you might need to update graphics drivers:

`sudo apt-get install libglvnd0 libglvnd-dev`

**Comment by kuangzhuoran** [comment]
And when I use the genome listed on gw, I met the same error

**Reply by kcleal** [reply]
Hi @kuangzhuoran,
You are using the correct command, but unfortunately you seem to be running into what's becoming a common error relating to openGL and GLFW. Could you share what system you are running, and how you installed gw? 
Thanks
